### ISSUE TITLE
Launcher appears to store all console logs in RAM

### ISSUE THREAD

**Issue by MoriyaShiine** [ATLauncher/ATLauncher#763]
### What happened?

1. Open ATLauncher
2. Launch an instance of Minecraft with extremely heavy log usage
3. See the launcher in Task Manager skyrocket in RAM usage

### Operating System

Linux

### Relevant log output

```shell
https://ufile.io/nahvy6se

Apologies for the unconventional upload, I could not find a pastebin-like site that would allow me to upload a 9mb file.
```

### Additional Information

![image](https://github.com/ATLauncher/ATLauncher/assets/45156123/4ff9af30-47db-4124-8773-609928e4866b)

This was done with an instance of Minecraft that outputs an absurd amount of `false`s to the log (issue with another mod, but relevant to the launcher). I've tested this with Prism Launcher, and the RAM usage stays consistent / does not increase in usage.

This isn't *too* bad of an issue with instances that have regular log outputs, but ideally if the console isn't visible then it shouldn't be using memory.

The name in the image just happens to be what Task Manager is showing; the console itself is not opened.

### Is this actually a launcher bug?

- [X] I am sure this is a bug with the launcher and not an issue with running Minecraft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by RyanTheAllmighty** [comment]
Yeah it's been this way for a long time now, it's never really been looked at though. #551 will have some part in fixing this going forward as logs will likely become similar to that of MultiMC and it's forks, where each launch has it's own console away from the launchers main console.

But in the meantime, worth seeing if there's anything that can be done now.

Out of interest @MoriyaShiine what pack, platform and pack version (or what mods are causing the log spam) so can replicate it?

**Comment by MoriyaShiine** [comment]
The pack is a custom one, but the only relevant mod is Effective 1.4 (Minecraft 1.19.2) since I haven't moved it from Fabric to Quilt yet. Running around in a world for a few minutes will populate the console with a bunch of `false` outputs.
